Realm/Realm/Schema/RealmSchema.cs

```diff
@@ -150,6 +152,35 @@ internal static RealmSchema CreateSchemaForClasses(IEnumerable<Type> classes, Re
             return builder.Build();
         }
 
+        internal static RealmSchema GetSchema(HashSet<Type> types)
```

Comment:
No need to pass `types` here - this function can operate on `_defaultTypes` directly.

Reply:
I agree with you, I just thought to follow the way it was done before. I'll update.